lego/apps/users/models.py

```diff
@@ -321,6 +322,16 @@ def restricted_lookup(self):
     def announcement_lookup(self):
         return [self]
 
+    def unanswered_surveys(self):
+        from lego.apps.surveys.models import Survey
+        from lego.apps.events.models import Registration
+        registrations = Registration.objects.filter(user_id=self.id, presence=PRESENT)
+        unanswered_surveys = Survey.objects.filter(
+            event__registrations__in=registrations, active_from__lte=timezone.now(),
+            template_type__isnull=True
+        ).exclude(submissions__user__in=[self]).prefetch_related('event__registrations')
```

Comment:
Adding `submissions__user` in the prefetch_related should reduce the queries required here, since this sumbissions has a foreignkey to Survey. Maybe you can profile this either using the django-debug-toolbar or profiling tool in pycharm to validate 👍